packages/aws-lambda/src/triggers.js

```diff
@@ -44,6 +44,9 @@ exports.enrichSpanWithTriggerData = function enrichSpanWithTriggerData(event, sp
     span.data.lambda.trigger = 'aws:sqs';
     extractEventFromSQS(event, span);
     return;
+  } else if (isInvokeFunction(context)) {
+    span.data.lambda.trigger = 'aws:sdk.invoke';
```

Comment:
I just double checked and Andrii added the trigger as `aws:lambda.invoke`. I wasn't aware of this earlier, but we should follow Go's example and use the same string. (That string is also a recognized trigger in our back end processing pipeline now.)

https://github.com/instana/go-sensor/blob/f06ddbbdb8958b0ed33a0c743e1cdbb97fe20b00/instrumentation/instalambda/handler.go#L162